js/apps/admin-ui/public/locales/en/identity-providers-help.json

```diff
@@ -38,6 +38,9 @@
   "trustEmail": "If enabled, email provided by this provider is not verified even if verification is enabled for the realm.",
   "accountLinkingOnly": "If true, users cannot log in through this provider.  They can only link to this provider.  This is useful if you don't want to allow login from the provider, but want to integrate with a provider",
   "hideOnLoginPage": "If hidden, login with this provider is possible only if requested explicitly, for example using the 'kc_idp_hint' parameter.",
+  "filteredByClaim": "If true, enables filter by claim, to reject authenticated users whose JWT claims do not include the configured claim filter",
```

Comment:
will do, even if it might change if we also include Stian's comment above:
> Perhaps it could be renamed to something like Require claim to authenticate and Required claim name / Required claim value?